readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://taninska.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](taninska.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Need to fix test report link
https://taninska.github.io/layout_search-bar-airbnb/report/html_report/